components/assetlist/stories/assetlist.stories.js

```diff
@@ -23,10 +32,10 @@ export default {
 		packageJson,
 		metadata,
 		status: {
-			type: "unmigrated",
+			type: "migrated",
 		},
 	},
-	tags: ["unmigrated"],
+	tags: ["migrated"],
```

Comment:
I think I asked this elsewhere as well, but are these considered migrated? I couldn't find a design spec (maybe there isn't one?) so couldn't verify.

Reply:
It's a contribution but since it's meeting all the S2 design standards, I think we can call this one good. Open to objections though of course!